### PR TITLE
Make Checkout and Client objects Arrayable

### PR DESCRIPTION
**Pull request by 4Giedrius**

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -443,4 +443,10 @@ public function setCreatedAt($created_at)
         return $this;
     }
 
+    public function toArray()
+    {
+        return get_object_vars($this);
```
Comment (fordnox) [inline src/Client.php]: cheater

Comment (vaidas-lungis) [inline src/Client.php]: giphy 1

**Comment by zygintas** [comment]
merge?

**Pull request merged**